test/csx.coffee

```diff
@@ -724,3 +692,11 @@ test 'unspaced less than after CSX works but is not encouraged', ->
 
       res = 2 < div;
     '''
+
+test 'CSX invalid attribute errors', ->
```

Comment:
All our error-checking tests are in `error_messages.coffee`. Perhaps they shouldn’t be, but that’s a refactor for another PR. Please move these checks over there and follow the form in that file, where you confirm that the error returned is as expected (i.e. not just that an exception is thrown).